pkg/selectors/kernel.go

```diff
@@ -601,8 +601,16 @@ func parseSelector(
 	return nil
 }
 
-// array := [number][filter1][filter2][...][filtern]
-// filter := [length][matchPIDs][matchBinaries][matchArgs][matchNamespaces][matchCapabilities][matchNamespaceChanges][matchCapabilityChanges]
+// array := [number][filter1_offset][filter2_offset][...][filtern_offset]
```

Comment:
Assuming array is the return value of `InitKernelSelectors()` shouldn't we have:
`array := [number][filter1_offset][filter2_offset][...][filtern_offset][filter1][filter2][...][filtern]`
after the offsets don't we have the actual filters?

Reply:
Yap, good point.

Reply:
Changed it to:
```
// [number][filter1_offset][filter2_offset][...][filtern_offset][filter1][filter2][...][filtern]
```